src/nomadic/backup/commands.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import subprocess
+from pathlib import Path
+
+import click
+
+from nomadic.util.dirs import produce_dir
+from nomadic.util.logging_config import LoggingFascade
+from nomadic.util.rsync import selective_rsync
+
+script_dir = Path(__file__).parent.resolve()
+log = logging.getLogger(script_dir.stem)
+
+
+@click.command(short_help="Backup sequence data folder to a local hard disk drive")
+@click.option(
+    "-b",
+    "--backup_dir",
```

Comment:
The common convention afaik is hyphens, so let's use them. Advantage is also easier typing, no need to press shift
```suggestion
    "--backup-dir",
```

Reply:
OK will leave as is